P5/Source/Specs/sex.xml

```diff
@@ -32,74 +32,55 @@ $Id$
       <datatype>
```

Comment:
Code from lines 31-35 should be deleted if no sample values are suggested.

Reply:
Fixed with https://github.com/TEIC/TEI/pull/2299/commits/1bd0f6a5ffa6712e20c0e545881b5755eff0073b